source/dynamical_systems/test/tests/test_point_attractor.cpp

```diff
@@ -23,6 +25,10 @@ class LinearDSTest : public testing::Test {
     std::cout << abs(current_pose.get_orientation().dot(target_pose.get_orientation())) << std::endl;
   }
 
+  std::shared_ptr<IDynamicalSystem<CartesianState>>
+      ds = DynamicalSystemFactory<CartesianState>::create_dynamical_system(
+      DynamicalSystemFactory<CartesianState>::DYNAMICAL_SYSTEM::POINT_ATTRACTOR
+  );
```

Comment:
it's a small thing, but you should do the `ds = DynamicalSystemFactory<CartesianState>::create_dynamical_system(DynamicalSystemFactory<CartesianState>::DYNAMICAL_SYSTEM::POINT_ATTRACTOR);` in the `SetUp` method. Default values for class properties are fine when they are literals (like `100`) or simple constructors, but here the constructor method is a bit complex and so is better suited for the SetUp call.